### PR TITLE
Update upgrading docs for Archivematica 1.12

### PR DESCRIPTION
**Pull request by tw4l**
- Remove 1.11-specific section on clearing watched directory
- Add section on updating Elasticsearch indices for 1.12

Connected to https://github.com/archivematica/Issues/issues/1257

### CODE REVIEW

File: admin-manual/installation-setup/upgrading/upgrading.rst [hunk 1]
```diff
@@ -393,6 +364,112 @@ set environment variables and restart Archivematica processes.
...
+   locations may require use of the option to rebuild indices from the Storage
+   Service API rather than from the filesystem. At this time, it is not
+   possible to rebuild the indices for all types of remote locations.
+
+.. note::
+   Please note, the execution of this command may take a long time for big
+   AIP and Transfer Backlog storage locations, especially if the packages are
+   stored compressed or encrypted, or you are using a third party service. If
+   that is the case, you may want to consider using the
+   :ref:`Update the ElasticSearch mappings <update-mappings>` approach instead.
```
Comment (sevein) [inline admin-manual/installation-setup/upgrading/upgrading.rst]: It should read "Elasticsearch" not "ElasticSearch".

File: admin-manual/installation-setup/upgrading/upgrading.rst [hunk 2]
```diff
@@ -6,47 +6,18 @@ Upgrade from Archivematica |previous_version|.x to |release|
...
-   processing the transfers/ingests before proceeding.
-
-2. Delete all contents of the completedTransfers watched directory.
-
-   .. code:: bash
-
-      sudo rm -rf /var/archivematica/sharedDirectory/watchedDirectories/SIPCreation/completedTransfers/*
-
-3. Perform the upgrade as described below.
-
```
Comment (sevein) [inline admin-manual/installation-setup/upgrading/upgrading.rst]: I think this is still relevant if you are upgrading from 1.10 or older, isn't it?

**Review by sevein — APPROVED** [review]
Super clear!

File: admin-manual/installation-setup/upgrading/upgrading.rst (hunk 2 above)
Comment (tw4l) [reply admin-manual/installation-setup/upgrading/upgrading.rst]: I was assuming that the instructions assumed users should do a stepped upgrade if jumping between versions, where they upgrade one version at a time. Not sure where I got that - maybe we used to recommend it? Happy to add it back if that's mistaken or we just want to err on the safe side.

**Comment by tw4l** [comment]
Thanks @sevein ! Should be good for re-review now.

File: admin-manual/installation-setup/upgrading/upgrading.rst (hunk 2 above)
Comment (sevein) [reply admin-manual/installation-setup/upgrading/upgrading.rst]: I can't think of a reason why a stepped upgrade would be needed. @sromkey what do you think?

**Review by sevein — APPROVED** [review]

File: admin-manual/installation-setup/upgrading/upgrading.rst (hunk 2 above)
Comment (sromkey) [reply admin-manual/installation-setup/upgrading/upgrading.rst]: I have no idea! :P

Comment (tw4l) [reply admin-manual/installation-setup/upgrading/upgrading.rst]: Hm, maybe I just made that up. Sounds like we should keep the completed transfers section in then, at least for a few versions!

**Pull request merged**